pkg/security/podsecuritypolicy/seccomp/strategy_test.go

```diff
@@ -102,51 +106,75 @@ func TestNewStrategy(t *testing.T) {
 }
 
 func TestGenerate(t *testing.T) {
+	bar := "bar"
 	tests := map[string]struct {
 		pspAnnotations  map[string]string
 		podAnnotations  map[string]string
+		seccompProfile  *api.SeccompProfile
 		expectedProfile string
 	}{
 		"no seccomp, no pod annotations": {
 			pspAnnotations:  withoutSeccomp,
 			podAnnotations:  nil,
+			seccompProfile:  nil,
```

Comment:
this is nil by default, can omit lines like this

Reply:
Fixed :+1: